EssentialsChat/src/com/earth2me/essentials/chat/EssentialsChatPlayer.java

```diff
@@ -38,9 +38,9 @@ String getChatType(final String message) {
 
         final char prefix = message.charAt(0);
         if (prefix == ess.getSettings().getChatShout()) {
-            return "shout";
+            return message.length() > 1 ? "shout" : "";
```

Comment:
```suggestion
!message.isEmpty() ? "shout" : "";
```
See this for proof: https://stackoverflow.com/a/19126404

Reply:
`message` here includes the prefix character, which means a valid shout needs to be at least two characters total, since this gets removed later.